addon/components/base/bs-nav.js

```diff
@@ -16,14 +16,14 @@ import layout from 'ember-bootstrap/templates/components/bs-nav';
  * [`nav.dropdown`](Components.Dropdown.html)
 
  ```hbs
- {{#bs-nav type="pills" as |nav|}}
-   {{#nav.item}}
-      {{#nav.link-to "foo"}}Foo{{/nav.link-to}}
-   {{/nav.item}}
-   {{#nav.item}}
-     {{#nav.link-to "bar"}}Bar{{/nav.link-to}}
-   {{/nav.item}}
- {{/bs-nav}}
+ <BsNav @type="pills" as |Nav|>
+  <Nav.item>
+    <Nav.link-to "foo">Foo</Nav.link-to>
```

Comment:
`Nav.link-to` seems wrong. Is it working? Do we need to put an alias there to support `<Nav.linkTo>`?

Reply:
I think it works as-is, but maybe linkTo is a good idea

Reply:
I think it's fine! We could provide an alias, but that's out of scope for this PR I think.